### PR TITLE
ui: add MVCC range stats to Db Console

### PR DESCRIPTION
**Pull request by koorosh**
This patch extends node details with `rangeKeyBytes`
and `rangeValueBytes` stats. Also, it adds the same
stats to Advanced Debug > Range Status section.

Release note (ui change): Added "Range Key Bytes" and
"Range Value Bytes" stats on Node details page.

### CODE REVIEW

File: pkg/ui/workspaces/db-console/src/views/cluster/containers/nodeOverview/tooltips.tsx [hunk 1]
```diff
@@ -85,6 +85,47 @@ export const ValueBytesTooltip: React.FC<
...
+  nodeName: string;
+}
+  > = props => (
+  <Tooltip
+    {...props}
+    placement="bottom"
+    title={
+      <div className="tooltip__table--title">
+        <p>
+          Number of bytes stored in range values on node{" "}
```
Comment (erikgrinaker) [inline pkg/ui/workspaces/db-console/src/views/cluster/containers/nodeOverview/tooltips.tsx:119]: nit: we should use "MVCC range key values" (and similarly "MVCC range keys" above), to disambiguate them. "range keys" or "range values" can sound like e.g. "keys in the Raft range", which is a different thing.

**Review by erikgrinaker — APPROVED** [review]
LGTM at a high level. This will handle missing fields from 22.1 nodes gracefully?

File: pkg/ui/workspaces/db-console/src/views/cluster/containers/nodeOverview/tooltips.tsx (hunk 1 above)
Comment (erikgrinaker) [reply pkg/ui/workspaces/db-console/src/views/cluster/containers/nodeOverview/tooltips.tsx]: Thanks!

**Comment by koorosh** [comment]
> LGTM at a high level. This will handle missing fields from 22.1 nodes gracefully?

Changed access to fields with fallback values in case they're not available for any reason.

Regarding version 22.1, this version serves it's own version of DbConsole that doesn't have this change (unless it is not backported) and relies on protobuf definitions without new stats. 
If user accesses DbConsole from 22.2 version, then it still have this field in proto message with `null` value for missing data. This case should be handled by added change with fallback values.

**Comment by erikgrinaker** [comment]
Makes sense, thanks! Wasn't sure if DB Console would receive data from other nodes -- if it only ever communicates with the local node then it should be fine. But good to have a fallback just in case.

Thanks again for getting this in so quickly!

**Pull request merged**